game-app/game-core/src/main/java/games/strategy/triplea/ui/EditPanel.java

```diff
@@ -490,33 +494,40 @@ public void actionPerformed(final ActionEvent event) {
             setWidgetActivation();
             final PlayerChooser playerChooser =
                 new PlayerChooser(getData().getPlayerList(), getMap().getUiContext(), false);
+            final Container parent = getTopLevelAncestor();
             final JDialog dialog =
-                playerChooser.createDialog(getTopLevelAncestor(), "Select owner PUs to change");
+                playerChooser.createDialog(parent, "Select Owner PUs/Resources to Change");
```

Comment:
What would you think if the title of the dialog were a more simple "Change Resources"?

Reply:
SG.